cmd/cycloid/infrapolicies/create.go

```diff
@@ -0,0 +1,107 @@
+package infrapolicies
+
+import (
+	"github.com/cycloidio/cycloid-cli/cmd/cycloid/common"
+	"github.com/cycloidio/cycloid-cli/cmd/cycloid/internal"
+	"github.com/cycloidio/cycloid-cli/cmd/cycloid/middleware"
+	"github.com/cycloidio/cycloid-cli/printer"
+	"github.com/cycloidio/cycloid-cli/printer/factory"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+// NewCreateCommand returns the cobra command
+// to create a new infrapolicy using a file
+// Note! For boolean flags it is required var=bool
+//       https://github.com/spf13/cobra/issues/613
+func NewCreateCommand() *cobra.Command {
+
+	var cmd = &cobra.Command{
+		Use:   "create",
+		Short: "create a infrapolicy",
+		Example: `
+	# create a infrapolicy my-policy
+	cy --org my-org ip create \
+	   --policy-path /path/to/file/file.rego \
+	   --name my-policy
+	   --description "an awesome infrapolicy" \
+	   --owner user_cannonical \
+	   --severity "advisory" \
+	   --enabled=true
+		`,
+		RunE:    create,
+		PreRunE: internal.CheckAPIAndCLIVersion,
+	}
+	common.RequiredFlag(WithFlagPolicyPath, cmd)
+	common.RequiredFlag(WithFlagName, cmd)
+	common.RequiredFlag(WithFlagOwner, cmd)
```

Comment:
Is this flag is really required ?
On some others entities, if it stay empty the BE will put the owner of the API key as owner by default

Reply:
On the current implementation flagowner is required, because there's a validation performed on the number of caracters